### PR TITLE
Dashboard count of host and participant fixed

### LINKED ISSUE #2641: Admin Dashboard don't give correct count of host and participant teams
After adding 3 host teams and two participant teams:
- the expected behaviour:
The count should have been 3 for host team and 2 for participant team
- the observed behaviour: The count showed 2 for both of them

Screen record link:
https://drive.google.com/file/d/1mwk7XcQ8xQm6pwsx5uwBrVpWVtyTxGUh/view?usp=sharing

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
Fix #2641 
Count of host and participant on the Admin Dashboard corrected.

### CODE REVIEW

**Comment by Suryansh5545** [comment]
@Kajol-Kumari Please fix the build

**Comment by Kajol-Kumari** [comment]
@Suryansh5545  Thank you for guidance.I made the required changes.

**Comment by Suryansh5545** [comment]
@Kajol-Kumari the build is still failing please take a look

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ describe('Unit tests for dashboard controller', function () {
                 },
             ];
             vm = createController();
-            expect(vm.hostTeamCount).toEqual(successResponse.count);
+            expect(vm.hostTeamCount).toEqual(successResponse.results.length);
```
Comment (nsjcorps) [inline frontend/tests/controllers-test/dashCtrl.test.js]: This is your issue. Change this to successResponse.length

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 2]
```diff
@@ -235,7 +235,7 @@ describe('Unit tests for dashboard controller', function () {
                 },
             ];
             vm = createController();
-            expect(vm.participatedTeamCount).toEqual(successResponse.count);
+            expect(vm.participatedTeamCount).toEqual(successResponse.results.length);
```
Comment (nsjcorps) [inline frontend/tests/controllers-test/dashCtrl.test.js]: Same here @Kajol-Kumari

File: frontend/tests/controllers-test/dashCtrl.test.js (hunk 1 above)
Comment (nsjcorps) [reply frontend/tests/controllers-test/dashCtrl.test.js]: There is no property of `results` in the `successResponse` array

Comment (Kajol-Kumari) [reply frontend/tests/controllers-test/dashCtrl.test.js]: @nsjcorps when i had done console log of "response.data" in dadhCtrl,results property was giving me the list of the array containing the data.

**Comment by Suryansh5545** [comment]
@Kajol-Kumari add results to successresponse like line 117 maybe that works and then you can use results.length

**Comment by Suryansh5545** [comment]
@Kajol-Kumari LGTM:+1:

**Comment by Kajol-Kumari** [comment]
Done! Thanks @Suryansh5545  and @nsjcorps for pointing out the problems

**Comment by Suryansh5545** [comment]
@Ram81 @deshraj @vkartik97 Please take a look

**Pull request merged**